packages/block-library/src/navigation-menu/edit.js

```diff
@@ -53,6 +56,7 @@ function NavigationMenu( {
 			<div className="wp-block-navigation-menu">
 				<InnerBlocks
 					allowedBlocks={ [ 'core/navigation-menu-item' ] }
+					moverOptions={ moverOptions }
```

Comment:
I think for now we are still exploring an API so the public interfaces we add should be marked as __experimental.
Maybe later we decide to just have a property isHorizontal that applies to movers drag & drop etc... Marking things as experimental leaves the door open.

Reply:
Agree that we should prefix these props `__experimental` while we're figuring out the best API.

Reply:
you are both right done :)